src/components/znkTimeline/timeline.service.js

```diff
@@ -25,11 +25,13 @@
             }
             if (ExerciseTypeEnum.SECTION) {
                 imgObj[ExerciseTypeEnum.SECTION.enum] = {icon: svgMap.section};
+                imgObj[ExerciseTypeEnum.SECTION.val] = {icon: svgMap.practice};
```

Comment:
save section val in the imageObj. to not be tied to practice, because we need both section diagnostic and regular section images for the timeline, the practice image is the section (not diagnostic) image.